packages/examples/src/api/fhir/operations/bot-deploy.ts

```diff
@@ -0,0 +1,21 @@
+import { MedplumClient } from '@medplum/core';
+
+const medplum = new MedplumClient();
+
+// start-block deployTs
+await medplum.post(medplum.fhirUrl('Bot', '[id]', '$deploy').toString(), { foo: 'bar' });
```

Comment:
Bot deploy takes 2 very specific body parameters: 
`filename`
`code`.

The docs should reflect this, instead of `{foo: 'bar'}`